pkg/cloud/alibaba/alibaba_test.go

```diff
@@ -0,0 +1,24 @@
+package alibaba
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestGetResources(t *testing.T) {
+	resources := GetResources()
+	assert.Len(t, resources, 2)
+
+	var names, kinds []string
+	for _, r := range resources {
+		names = append(names, r.GetName())
+		kinds = append(kinds, r.GetObjectKind().GroupVersionKind().Kind)
+	}
+
+	assert.Contains(t, names, "alibaba-cloud-controller-manager")
+	assert.Contains(t, kinds, "Deployment")
+
+	assert.Contains(t, names, "alibaba-cloud-controller-manager-config")
+	assert.Contains(t, kinds, "ConfigMap")
```

Comment:
Doesn't the installer normally create the cloud config? Is there WIP for that already? 

If they are creating it, then we have a mechanism in place already to copy this over and make it available to the CCM, so it wouldn't be needed here

Reply:
The installer does not create the cloud config, and does not plan to do so. OpenStack also contains a configmap. Can we add the cloud config here?

Reply:
As far as I can tell the installer does still create the cloud config, based on this code https://github.com/openshift/installer/blob/af373729b31fcc6a5af9b6d0a708a44dc2ca1cb2/pkg/asset/manifests/cloudproviderconfig.go#L91-L204

Have you spoken with the installer team about this already? If so, let me know where so I can grab that extra context

Reply:
Looking at openshift/installer#5018: Alibaba is currently creating an empty cloud provider config: https://github.com/openshift/installer/blob/c8edf03a275d5025efaf375f3c04499681a61901/pkg/asset/manifests/alibabacloud/cloudproviderconfig.go

If there are values needed from the cloud provider config, they should be generated in the installer.